evm/hardhat.config.js

```diff
@@ -1,5 +1,13 @@
 require("@nomicfoundation/hardhat-toolbox");
 
+const SEPOLIA_PRIVATE_KEY = "b8d88a575069fd56f60618bceb68ca30e8643e948ed9cf8e82230b31edc47269";
```

Comment:
Whose secret is this? Also, it's not a secret anymore, it's a leaked secret. :P

Reply:
My bad, thanks. 
I changed this to using `dotenv-flow`.